00-Install_and_Setup/README.md

```diff
@@ -1,77 +1,125 @@
 # Getting Started with Python and Anaconda
 
 Please come to the workshop with a laptop already configured as described below.
-If you have any problems with any of these steps, please [open an issue](https://github.com/astropy/astropy-workshop/issues).
+If you have any problems with any of these steps, please
+[open an issue](https://github.com/astropy/astropy-workshop/issues).
 
 ## 1. Clone This Repository
 
-Download the workshop folder using [git](https://help.github.com/articles/set-up-git/):
+Download the workshop folder using
+[git](https://help.github.com/articles/set-up-git/):
 
     % git clone https://github.com/astropy/astropy-workshop.git
 
-If you don't have git installed, you can download the ZIP file by pressing the green *Clone or download* button at https://github.com/astropy/astropy-workshop and selecting *Download ZIP*.  However, this option is less prefered because it makes it more difficult for you to update your copy of the repository if updates are made.
+If you don't have git installed, you can download the ZIP file by pressing the
+green *Clone or download* button at
+https://github.com/astropy/astropy-workshop and selecting *Download ZIP*.
+However, this option is not recommended because it impedes the ability to
+update your copy of the repository if updates are made.
 
-## 2. Install Anaconda (if needed)
+## 2. Install Miniconda (if needed)
 
-*Anaconda includes conda, conda build, Python and 100+ automatically installed, open source scientific packages and their dependencies that have been tested to work well together, including SciPy, NumPy and many others.*
+*Miniconda is a free minimal installer for conda. It is a small, bootstrap
+version of Anaconda that includes only conda, Python, the packages they depend
```

Comment:
(note somewhere in here that anaconda can be used if already installed)

Reply:
Is this already handled by your suggestion at a8ecb5a3108fb194f01a3c438f974ec1e9943b17 ?

Reply:
I think it's covered.